### PR TITLE
made developer list "neutral" and added an id field

### LINKED ISSUE #9: Missing developer IDs result in bnd-maven-plugin warnings
The bnd-maven-plugin tries to use the developer information in the openhab-super-pom for deriving bundle manifest headers.

Apparently the POM is missing the developer IDs which results in lots of warnings that show in the Eclipse "Error Log" view:

error-log

The complete error message is:

`
Cannot consider developer in line '32' of file '/home/wouter/.m2/repository/org/openhab/openhab-super-pom/1.0.0/openhab-super-pom-1.0.0.pom' for bundle header 'Bundle-Developers' as it does not contain the mandatory id.`

Adding an `<id>` tag for each developer in the super POM will probably fix the warning. However that will probably also add Kai and Patrick as developers in the bundle manifest headers of each bundle.

What do you think would be the best solution?

See also:

* [Developers in openhab-super-pom](https://github.com/openhab/infrastructure/blob/de07850a121c754e6fde386c2ab34815d2de55de/pom.xml#L25-L38)
* [Developers (Maven POM Reference)](https://maven.apache.org/pom.html#Developers)
* [BndMavenPlugin.java code](https://github.com/bndtools/bnd/blob/aa80c6474da94fe4fbd3e50fc361c52a887fb3fe/maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java#L378-L404)
* [bnd issue for adding this logic](https://github.com/bndtools/bnd/issues/2447)

### PR DESCRIPTION
**Pull request by kaikreuzer**
Fixes #9

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,14 +24,9 @@
...
-      <organizationUrl>http://www.openHAB.org</organizationUrl>
-    </developer>
-    <developer>
-      <name>Kai Kreuzer</name>
-      <email>kai@openhab.org</email>
+      <id>openhab</id>
+      <name>openHAB</name>
+      <email>info@openhab.org</email>
       <organization>openHAB</organization>
       <organizationUrl>http://www.openHAB.org</organizationUrl>
```
Comment (maggu2810) [inline pom.xml]: Should we change that to `https` while touching it?

Comment (pfink) [reply pom.xml]: makes sense

Comment (kaikreuzer) [reply pom.xml]: done.

**Review by maggu2810 — APPROVED** [review]

**Review by wborn — APPROVED** [review]

**Comment by pfink** [comment]
Thanks to everyone!

**Comment by kaikreuzer** [comment]
Version 1.0.1 has been published to Maven Central.

**Pull request merged**